magicctapipe/scripts/lst1_magic/magic_calib_to_dl1.py

```diff
@@ -308,6 +345,25 @@ def magic_calib_to_dl1(input_file, output_dir, config, max_events, process_run=F
             elif tel_id == 2:
                 event_info.tel_id = config["mc_tel_ids"]["MAGIC-II"]  # MAGIC-II
 
+            if event.trigger.tels_with_trigger == [1, 2]:
```

Comment:
this conversion here from the old numbering to the new numbering is really confusing. It needs at least some explanations.

Reply:
It is not a new numbering. Anyway, if one uses 4LSTs, this will not work (realized just now, because I was focused in the 3 real telescopes case only). As we discussed via email, also here we can use the binary format stored as int, so that it will be unique for any telescope combination.

Reply:
ok, again let's merge it as it is, but we can simplify the code in another PR (just a single loop over tels_with_trigger and converting them into assigned_tel_ids)